sound/soc/sof/intel/hda.c

```diff
@@ -63,6 +63,7 @@ static int sdw_params_stream(struct device *dev,
 
 	data.dai_index = (params_data->link_id << 8) | d->id;
 	data.dai_data = params_data->alh_stream_id;
+	data.dai_node_id = data.dai_data;
```

Comment:
@plbossart the series looks good but just have a question about why we need to set dai_data at all here if it is not used?

Reply:
@ranj063 data_data is used by IPC3 solutions. we can't remove it and I don't want to special case this part.

Reply:
Ahh i see, thanks!